script/dev/start-server.sh

```diff
@@ -13,14 +13,6 @@ if [[ "$PWD" != "$HOME"* ]]; then
   exit 1;
 fi
 
-ORCH_URL_ROOT=${ORCH_URL_ROOT:-'https://firecloud-orchestration.dsde-dev.broadinstitute.org'}
+docker run --rm -it -v "$PWD":/working broadinstitute/dsde-toolbox render-templates.sh local $VAULT_TOKEN
```

Comment:
I was hoping to convince you that it made more sense to make this project look like the others, push existing src sub dirs down into /src/main. There is nothing scala-esqe about it and it leaves room for other types of src like test.

plus I think the render-templates script you wrote assumes this structure so it makes it easy if all the projects are the same.